portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function randomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = Fact;
 }
+function getName(){
+    
+     fetch('/data').then(response => response.json()).then((list) => {
+         console.log(list[0])
+    const json = document.getElementById('list-container');
+    json.innerHTML = '';
+    json.appendChild(
+        createListElement('1: ' + list[0]));
+    json.appendChild(
+        createListElement('2: ' + list[1]));
+    json.appendChild(
+        createListElement('3: ' + list[2]));
+
+  });
+}
+function createListElement(text) {
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace): Please add a blank line between the two functions

```suggestion

function createListElement(text) {
```